python3eval/pythoneval.cs

```diff
@@ -63,6 +65,17 @@ public override object Evaluate(string code, IList bindingNames, IList bindingVa
             if (code != prev_code)
             {
                 var pythonEngine = Python.CreateEngine();
+                //to maintain compatability with ironPython code written in .netframework - we load the system.dll shim
+                //which loads many types which used to be in system.dll(mscorlib) like system.diagnostics.process
+                //which were moved in .Net. These types are now importable by python code without users needing to add
+                //clr.addreference.
+                //TODO consider a preference for this.
+                //TODO test smaller shims...netstd.dll
+               
+                if (CONFIG_ENABLE_NET48SHIMCOMPAT)
+                {
+                    pythonEngine.Runtime.LoadAssembly(Assembly.Load("System"));
```

Comment:
And you're not making this the default because of performance overhead?

Reply:
I actually did make it the default (it's true in the config file at the moment but that was up for debate) - but yes I am introducing this config option for performance reasons.

Reply:
Ignorant question - Is the dll named "system.dll" a shim dll for mscorlib.dll?

Reply:
I think system.dll is coming from the old ASP.Core framework
mscrolib.dll is coming from the old netframework

Reply:
not quite.
https://stackoverflow.com/questions/402582/mscorlib-dll-system-dll#:~:text=dll%20itself%20doesn%27t%20contain,any%20calls%20into%20the%20runtime).